docusaurus/src/css/custom.css

```diff
@@ -88,7 +95,7 @@ img {
 }
 
 .menu__link--sublist-caret:after {
-    background: var(--ifm-menu-link-sublist-icon) 50%/1.5rem 1.5rem;
+    background: var(--ifm-menu-link-sublist-icon) 50%/2rem 2rem;
```

Comment:
:information_source: Make sure the carets are the same size as the carets defined in the theme.